### PR TITLE
Show recording icon in mobile

### PR DESCRIPTION
**Pull request by GhaziTriki**
- Show recording icon in mobile
- Fix message when a new message if the meeting id does not exist.
- Show a new message if the application was not invoked from a browser link.

### CODE REVIEW

File: clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/user/services/UsersMessageReceiver.as [hunk 1]
```diff
@@ -172,16 +173,14 @@ package org.bigbluebutton.lib.user.services {
 			userSession.joinMeetingResponse(msg);
 		}
 		
-		private function handleRecordingStatusChanged(m:Object):void {
-			var msg:Object = JSON.parse(m.msg);
+		private function handleRecordingStatusChanged(msg:Object):void {
```
Comment (capilkey) [inline clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/user/services/UsersMessageReceiver.as]: All of the message handler function have the name "handle<MessageName>". Can you please change this to match.

File: clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/user/services/UsersMessageReceiver.as [hunk 2]
```diff
@@ -172,16 +173,14 @@ package org.bigbluebutton.lib.user.services {
...
 			trace(LOG + "handleRecordingStatusChanged() -- recording status changed");
-			userSession.recordingStatusChanged(msg.recording);
+			meetingData.meetingStatus.recording = msg.body.recording;
 		}
 		
-		private function handleGetRecordingStatusReply(m:Object):void {
+		private function handleGetRecordingStatusReply(msg:Object):void {
```
Comment (capilkey) [inline clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/user/services/UsersMessageReceiver.as]: All of the message handler function have the name "handle<MessageName>". Can you please change this to match.

File: clients/flash/air-client/src/org/bigbluebutton/air/main/models/UISession.as [hunk 3]
```diff
@@ -1,91 +1,97 @@
...
-		public function get currentPageDetails():Object {
-			var details:Object = null;
-			if (_listPages.length > 0) {
-				details = _listPages.getItemAt(_listPages.length - 1).details;
-			}
-			return details;
-		}
-	}
-}
+package org.bigbluebutton.air.main.models {
```
Comment (capilkey) [inline clients/flash/air-client/src/org/bigbluebutton/air/main/models/UISession.as]: Please fix this file so it doesn't say that the whole file has been edited.

Comment (GhaziTriki) [reply clients/flash/air-client/src/org/bigbluebutton/air/main/models/UISession.as]: It is an EOL issue. I fixed it in cac2ce79901f2fe9446be2f54ad459d38029ad73, but it seems not te be fixed.

**Pull request merged**